python/snewpy/models/ccsn.py

```diff
@@ -425,7 +429,7 @@ class Fornax_2022(loaders.Fornax_2022):
         """
     def __init__(self, progenitor:str):
         self.metadata['Progenitor mass']=(float(progenitor[:-3]) if progenitor.endswith('bh') else float(progenitor)) * u.Msun
-
+        self.metadata['Black hole'] = progenitor.endswith('bh')
```

Comment:
I like this way of tackling the problem that was raised about having a second parameter due to the "bh" for some masses